unblob/cli.py

```diff
@@ -453,35 +464,36 @@ def print_report(reports: ProcessResult):
             valid_size += size
         total_size += size
 
-    if total_size == 0:
-        return
+    console = Console()
 
-    summary = Panel(
-        f"""Extracted files: [#00FFC8]{total_files}[/#00FFC8]
+    if total_size:
+        summary = Panel(
+            f"""Extracted files: [#00FFC8]{total_files}[/#00FFC8]
```

Comment:
I would put the output path in the summary panel, first line.

Reply:
Cool! Done.